collective_coverage.py

```diff
@@ -0,0 +1,89 @@
+import inspect
+import getopt
+import sys
+import os
+
+
+def get_full_path(filename):
+    current_file_path = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
```

Comment:
With python 3, i think `pathlib` and `__file__` is the preferred way of doing this. See [this](https://stackoverflow.com/questions/3430372/how-do-i-get-the-full-path-of-the-current-files-directory) stackoverflow question.